src/pages/studyView/addChartButton/customCaseSelection/CustomCaseSelection.tsx

```diff
@@ -25,6 +27,7 @@ export interface ICustomCaseSelectionProps {
     onSubmit: (chart: NewChart) => void;
     queriedStudies: string[];
     disableGrouping?: boolean;
+    disableChartTypeDropdown?: boolean;
```

Comment:
I don't think you need this option at the moment. It could totally rely on the property `disableGrouping `